C99 standard required but not enforced

I am unable to build HSTCAL from source on my machine. It got this error:
```
../pkg/acs/calacs/acs2d/dodark.c:161:14: error: ‘for’ loop initial declarations are only allowed in C99 mode
              for (unsigned int i = 0; i < numSupportedSubApertures; i++) {
```

@jhunkeler said there should have been some sort of `-std=c99` in a `wscript` somewhere but I don't see it.

The commands I used:
```
export PKG_CONFIG_PATH=.../miniconda/envs/py37/lib/pkgconfig

./waf configure --prefix=.../miniconda/envs/py37

./waf build -vvv
```

--- a/wscript
+++ b/wscript
@@ -338,6 +338,9 @@ def configure(conf):
 
     _determine_sizeof_int(conf)
 
+    if conf.check_cc(cflags='-std=c99'):
+        conf.env.append_value('CFLAGS', '-std=c99')
+
     # check whether the compiler supports -02 and add it to CFLAGS if it does
     if conf.options.debug:
         if conf.check_cc(cflags='-g'):

